src/app.js

```diff
@@ -132,6 +133,15 @@ var app = function(options) {
     });
   }
 
+  if (options.rootDocsRedirect) {
+    let link = options.docs.documenter.getDocumentationUrl();
+    DOCS_HTML = `<html><body><p>You're lost, here's the way out.</p>
+    <a href= ${link} >Refer to the documentation</a></body></html>`;
```

Comment:
I believe you'll want double quotes around `${link}`. It will end up looking like `...href="${link}"`